### PR TITLE
Simplify helper methods to closer align to API for select options

### PR DESCRIPTION
**Pull request by brilee**

### CODE REVIEW

**Comment by brilee** [comment]
`_get_selection' and `_create_where` are going to be replaced by the unified select api; this aligns the existing implementations closer to what the select API will want.

File: lilac/data/dataset_duckdb.py [hunk 1]
```diff
@@ -2485,36 +2477,49 @@ def _path_to_col(self, path: Path, quote_each_part: bool = True) -> str:
...
     if include_labels:
       include_filters = [Filter(path=(label,), op='exists') for label in include_labels]
       include_labels_query = f"({' OR '.join(self._create_where(manifest, include_filters))})"
+      filter_sql.append(include_labels_query)
 
     for label in exclude_labels:
       filters.append(Filter(path=(label,), op='not_exists'))
+    return filters, filter_sql
+
+  def _get_selection(
```
Comment (nsthorat) [inline lilac/data/dataset_duckdb.py:2500]: optional: this may be the moment to start moving these to a util to break this file up?

**Review by nsthorat — APPROVED** [review]
wooooo

**Review by dsmilkov — APPROVED** [review]
nice

**Pull request merged**